### PR TITLE
docs(i18n): sync i18n files from crowdin

### PR DESCRIPTION
**Pull request by Jack-Works**

### CODE REVIEW

**Review by guanbinrui — APPROVED** [review]

**Comment by Jack-Works** [comment]
@guanbinrui @septs plz review. Now it can integrate with Crowdin well. And I have also fixed #3446

File: packages/dashboard/src/locales/index.ts [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+import zh_TW from './zh-TW.json'
+
 export * from './i18n_generated'
+export const languages = {
+    'en-US': en_US,
+    'ja-JP': ja_JP,
+    'ko-KR': ko_KR,
+    'qya-AA': qya_AA,
+    'zh-CN': zh_CN,
+    'zh-TW': zh_TW,
```
Comment (septs) [inline packages/dashboard/src/locales/index.ts:17]: plz add more locale support

#3933 #3927 #3926 #3925 #3924

Comment (Jack-Works) [reply packages/dashboard/src/locales/index.ts:17]: I'll add it after this PR.

**Comment by Jack-Works** [comment]
@guanbinrui Please pick this one to 1.36 version.

**Pull request merged**